python_modules/libraries/dagster-components/dagster_components/core/schema/base.py

```diff
@@ -3,19 +3,14 @@
 
 from pydantic import BaseModel, ConfigDict, TypeAdapter
 
-from dagster_components.core.schema.metadata import (
-    JSON_SCHEMA_EXTRA_DEFER_RENDERING_KEY,
-    get_resolution_metadata,
-)
+from dagster_components.core.schema.metadata import get_resolution_metadata
 from dagster_components.core.schema.resolver import TemplatedValueResolver
 
 
 class ComponentSchemaBaseModel(BaseModel):
     """Base class for models that are part of a component schema."""
 
-    model_config = ConfigDict(
-        json_schema_extra={JSON_SCHEMA_EXTRA_DEFER_RENDERING_KEY: True}, extra="forbid"
-    )
+    model_config = ConfigDict(extra="forbid")
```

Comment:
@benpankow you made this same change on one of your PRs for other (similar) reasons, duplicating it here